scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/simulation/AnchorSchedulerTest.java

```diff
@@ -215,16 +232,14 @@ public void activitiesAnchoredToPlan() throws SchedulingInterruptedException {
           simulatedActivities,
           Map.of(), //unfinished
           planStart,
-          tenDays, // simulation duration
+          tenDays.minus(Duration.of(9, Duration.MINUTE)), // simulation duration
```

Comment:
because we use CheckpointSimulationDriver.stopOnceAllActivitiesAreFinished() as a stop condition so it stops at the end of the last activity, not a the end of the plan horizon.

Reply:
Resolving this comment as the test `CheckpointSimulationFacadeTest.testStopsAtEndOfPlanningHorizon()` exists, which tests the root of what I was concerned about.